include/pros/rtos.hpp

```diff
@@ -320,6 +321,22 @@ class Task {
 	task_t task;
 };
 
+// STL Clock compliant clock
+class ProsClock {
+	public:
+	typedef uint32_t rep;
+	typedef std::milli period;
+	typedef std::chrono::duration<rep, period> duration;
+	typedef std::chrono::time_point<ProsClock> time_point;
```

Comment:
prefer `using` declarations (e.g. `using period = std::milli` [is that a typo?])